hubblestack/utils/systemd.py

```diff
@@ -0,0 +1,92 @@
+# -*- coding: utf-8 -*-
+'''
+Contains systemd related help files
+'''
+# import python libs
+from __future__ import absolute_import, print_function, unicode_literals
+import logging
+import os
+import re
+import subprocess
+
+# Import Salt libs
+from hubblestack.utils.exceptions import HubbleInvocationError
+import hubblestack.utils.stringutils
+
+log = logging.getLogger(__name__)
+
+
+def booted(context=None):
+    '''
+    Return True if the system was booted with systemd, False otherwise.  If the
+    loader context dict ``__context__`` is passed, this function will set the
+    ``salt.utils.systemd.booted`` key to represent if systemd is running and
+    keep the logic below from needing to be run again during the same salt run.
+    '''
+    contextkey = 'salt.utils.systemd.booted'
```

Comment:
it has to be "hubblestack.utils.systemd.booted". 
I added hubblestack.utils.systemd. So you don't have to add systemd.py file from salt..

Reply:
fixed